benchmark/benchmark.js

```diff
@@ -1,11 +1,15 @@
 import { default as React } from "react";
-import { renderToString as renderToStringReact } from "react-dom/server";
+import { renderToString } from "react-dom/server";
 import { range } from "lodash";
 
-import { renderToString } from "../src";
+import { render } from "../src";
 import { time } from "./_util";
 
 
+// Make sure React is in production mode.
+process.env.NODE_ENV = "production";
```

Comment:
`renderToString` performance was considerably different after I made this changed, compared to before.  I'm not entirely sure what do do with the `benchmark/` dir - its not all benchmarks, its really just a collection of manual tests that have helped me along...